pest/src/span.rs

```diff
@@ -265,6 +265,16 @@ mod tests {
     use alloc::borrow::ToOwned;
     use alloc::vec::Vec;
 
+    #[test]
```

Comment:
These changes seem to have snuck in? Not an issue though, I'm not strict about a perfectly clean git history.

Reply:
Oh yes, one extra test got in